examples/js/exporters/GLTFExporter.js

```diff
@@ -596,11 +596,11 @@ THREE.GLTFExporter.prototype = {
 			}
 
 			// alphaMode
-			if ( material.transparent ) {
+			if ( material.transparent || material.alphaTest > 0.0 ) {
 
-				gltfMaterial.alphaMode = 'MASK'; // @FIXME We should detect MASK or BLEND
+				gltfMaterial.alphaMode = material.opacity < 1.0 ? 'BLEND' : 'MASK';
```

Comment:
That should work for now, although `mask` and `blending` is something that probably should be improved on the glTF spec. For example if we have a material with `opacity: 1` and `transparent :true `, and a transparent png, and no `alphatest`, it will enable blending on three.js too, instead of just masking.

Reply:
What would be the purpose of `opacity:1` + `transparent:true`? Not sure I understand this example but please weigh in on https://github.com/KhronosGroup/glTF/issues/1158 if you have suggestions. 🙂

Reply:
`opacity: 1 && transparent: true` will still use the alpha channel of the map (if I remember it correctly), if `transparent: false` it won't.